index.html

```diff
@@ -5,3 +5,6 @@
 stuff
 
 ljksdfkusdfiuh
+
+
+jk;nsdkjnsdjkng
```

Comment:
This variable name is not very semantic, can you please rename to something better?